<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hadesvm::cereon::Kis1KeyboardEditor</class>
 <widget class="QWidget" name="hadesvm::cereon::Kis1KeyboardEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="controllerStatePortLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="controllerCompartmentComboBox">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>30</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="controllerCompartmentLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Compartment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="controllerStatePortLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>KIS1 STATE port (hex):</string>
   </property>
  </widget>
  <widget class="QLabel" name="layoutLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Layout:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="layoutComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>controllerStatePortLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>hadesvm::cereon::Kis1KeyboardEditor</receiver>
   <slot>_onControllerStatePortLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_onControllerStatePortLineEditTextChanged(QString)</slot>
 </slots>
</ui>
